build.gradle

```diff
@@ -1,3 +1,23 @@
+/*
+ * ========================LICENSE_START=================================
```

Comment:
To be fair, adding the license headers to all these files could have been done in a separate pull request.  That would make reviewing the changes easier.